client/state/preferences/schema.js

```diff
@@ -62,15 +62,15 @@ export const remoteValuesSchema = {
 		colorScheme: {
 			type: 'string',
 			enum: [
-				'default',
-				'light',
-				'dark',
 				'classic-blue',
 				'classic-bright',
-				'laser-black',
-				'powder-snow',
+				'contrast',
+				'midnight',
 				'nightfall',
+				'ocean',
+				'powder-snow',
 				'sakura',
+				'sunset',
```

Comment:
I created this list based on the values here:

https://github.com/Automattic/wp-calypso/blob/360ebd2ea5c7613a86250d8b95d2231452d19355/client/blocks/color-scheme-picker/constants.js#L9-L71

I have no idea whether this is the canonical list.

Reply:
This is the canonical list!

Reply:
I'll add a comment here to indicate that this list and the schema need to be kept in sync.